### PR TITLE
feature/an/cardGroup

### PR DESCRIPTION
**Pull request by nicolino1**
-added styles to CardGroup and Card components.
- Mobile and desktop views finished.

### CODE REVIEW

File: src/components/CardGroup/CardGroup.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.card-group__container {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    height: 100%;
+    width: 50%;
+    gap: 20px;
+
```
Comment (monsterkosh) [inline src/components/CardGroup/CardGroup.css:7]: delete empty line

File: src/components/Cards/Card.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+.card-small .card-main,
+.card-imageHover .card-main{
+    margin: 20px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.card-title {
+    font-size: 18px;
```
Comment (monsterkosh) [inline src/components/Cards/Card.css:41]: dont use px on font-size

File: src/components/Cards/Card.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    transform: scale(1.5);
+    cursor: pointer;
+  }
+
+  .card-details {
+    margin: 20px;
+  }
+
+  .card-currentTime {
+    font-size: 40px;
```
Comment (monsterkosh) [inline src/components/Cards/Card.css:59]: change px to rem

File: src/components/Cards/Card.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  .card-previousTime {
+    font-size: 14px;
```
Comment (monsterkosh) [inline src/components/Cards/Card.css:63]: px to rem

**Review by monsterkosh — CHANGES_REQUESTED** [review]
small changes to font size unit

**Review by Gastonben — COMMENTED** [review]
After revisions LGTM.

**Review by monsterkosh — APPROVED** [review]
LGTM

**Review by Gastonben — APPROVED** [review]
LGTM

**Pull request merged**